For torch::jit::module's attr method to moble::module

kimishpatel opened the pull request:
Summary: This diff adds attr getter to mobile::module similar to torchscript module at https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/api/object.h#L75-L83.

Test Plan: CI

Differential Revision: D24604950

iseeyuan commented:
Would be nice to have get_attr for mobile module. Could you add a unit test on it? I think we could use model similar to https://github.com/pytorch/pytorch/blob/master/test/cpp/jit/test_lite_interpreter.cpp#L51

kimishpatel committed:
For torch::jit::module's attr method to moble::module (#47059)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/47059

This diff adds attr getter to mobile::module similar to torchscript module at https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/api/object.h#L75-L83.

Test Plan: LiteInterpreterTest::CheckAttrAccess

Reviewed By: xta0

Differential Revision: D24604950

fbshipit-source-id: 829e9a9451b6bd12a291e1731feac14271196b37
